src/pages/commits/index.tsx

```diff
@@ -233,7 +235,10 @@ const Commits = () => {
             versions={versions}
             revision={revision}
             isLoading={
-              (loading && !versions) || !projectIdentifier || isResizing
+              (loading && !versions) ||
+              !projectIdentifier ||
+              isRedirecting ||
```

Comment:
[nit] Is this maybe overkill since we check `loading` for a function that is skipped if we're redirecting?

Reply:
good question, I checked it manually, and it looks like the screen will flicker without `isRedirecting` 😔